fhirstore/search/bundle.py

```diff
@@ -0,0 +1,53 @@
+from fhirstore.search import SearchArguments
+
+
+class Bundle:
+    def __init__(self):
```

Comment:
Don't know what you and @simonvadee think but maybe `formatting_args` could be an attribute as you give it to `initiate_bundle` and then to all the other methods without changing it.
Also, I find it a bit weird to have an empty `__init__` but another method called `initiate_...` maybe we could do
```
bundle = Bundle(formatting_args, resource_type)
```
instead of
```
bundle = Bundle()
bundle. initiate_bundle(formatting_args, resource_type)
```

Reply:
agreed

Reply:
done

Reply:
Now we do 
`bundle = Bundle()` and then `bundle.fill()` or `bundle.append()`